packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -36,7 +38,11 @@ pub struct Abigen {
 
 impl Abigen {
     /// Creates a new contract with the given ABI JSON source.
-    pub fn new<S: AsRef<str>>(contract_name: &str, abi_source: S) -> Result<Self, Error> {
+    pub fn new<S: AsRef<str>>(
+        contract_name: &str,
+        abi_source: S,
+        binary_filepath: &Option<String>,
```

Comment:
any special reason this isn't `Option<&str>` instead of `&Option<String>` ?

Reply:
adjusted it